spec/cypress/app_commands/scenarios/admin/suppliers.rb

```diff
@@ -7,3 +7,5 @@
 
 lead_provider_2 = FactoryBot.create(:lead_provider, cohorts: [cohort])
 ProviderRelationship.create!(delivery_partner: delivery_partner, lead_provider: lead_provider_2, cohort: cohort)
+
+FactoryBot.create(:user, :lead_provider, full_name: "John Wick", email: "john-wick@example.com")
```

Comment:
Running scenarios tend to obfuscate the data - we could convert those into steps given that using Factory bot is supported by cypress gem